app/controllers/api/schools_controller.rb

```diff
@@ -15,13 +15,14 @@ def show
     end
 
     def create
-      result = School::Create.call(school_params:)
+      @school = School.new(school_params)
```

Comment:
This isn't wrong, but by calling `.new` directly you're circumventing the library at `lib/concepts/school/operations/create.rb`, I think unless we're making a more generalised move towards this approach we should try and follow the existing pattern. 

I had also spotted that the library was eating errors, so surfacing them here will be good all round.

Reply:
> I think unless we're making a more generalised move towards this approach we should try and follow the existing pattern.

I agree 👍

@tuzz On the off-chance you're seeing this - had you thought about whether/how validation errors would be surfaced from the API?

Reply:
Oh good spot, I'll update the library to provide errored fields which should achieve the same effect

Reply:
Yep you did 😄 - https://github.com/RaspberryPiFoundation/editor-api/issues/244

Reply:
@tuzz 
> I copied the existing pattern with the plan to improve it later 

I see 👍